src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,75 @@
+import {useMemo, useState} from "react";
```

Comment:
Codacy has a fix for the issue: [Replace `useMemo,·useState` with `·useMemo,·useState·`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
import { useMemo, useState } from "react";
```